### PR TITLE
[miio] Update many channels with tags, categories & QuantityTypes

### PR DESCRIPTION
**Pull request by marcelrv**
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/database/huayi.light.fanwy2-miot.json [hunk 1]
```diff
@@ -192,7 +196,7 @@
 				"piid": 3,
 				"friendlyName": "Presets - Pre Brightness",
 				"channel": "pre-brightness",
-				"type": "Number:Dimensionless",
+				"type": "Dimmer",
 				"unit": "percentage",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/huayi.light.fanwy2-miot.json:200]: Are units still used if you use a Dimmer instead?

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/huayi.light.fanwy2-miot.json]: In my testing it does not seem to harm. 
But... I did notice when I use in the item formatting %unit% it does not accept that for a dimmer, 
so there is no value in having the unit specified. 
Hence I removed the units from the dimmers now.

**Comment by marcelrv** [comment]
Do you see any possibility to have this still in the build. 
I was hoping this could be in the first OH3 release so that when folks upgrade they have the new channels from the gekko.

**Comment by marcelrv** [comment]
Also it fixes all the value mappings in the readme which may cause some confusion

**Comment by cpmeister** [comment]
We are in a feature freeze for the new release. So we are only allowing bug fixes for now.

**Comment by marcelrv** [comment]
I'll separate the fix of readme items from this PR than to fix the documentation prior to the release

**Comment by marcelrv** [comment]
the bug part of this has now been separated out of this PR and entered as separate PR #9402.
Once that one is merged, I'll update this PR to fix the created automatic merge issue

**Comment by cpmeister** [comment]
Can you resolve the conflicts?

**Comment by marcelrv** [comment]
@cpmeister yes, they should be fixed now. I think it is good to go again

File: bundles/org.openhab.binding.miio/README.md [hunk 2]
```diff
@@ -378,86 +457,6 @@ Currently the miio binding supports more than 250 different models.
...
 
 
-# Advanced: Unsupported devices
-
-Newer devices may not yet be supported.
-However, many devices share large similarities with existing devices.
-The binding allows to try/test if your new device is working with database files of older devices as well.
-
-There are 2 ways to get unsupported devices working, by overriding the model with the model of a supported item or by test all known properties to see which are supported by your device.
-
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:388]: Are you changing the newline characters again?

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Review by cpmeister — APPROVED** [review]

File: bundles/org.openhab.binding.miio/README.md (hunk 2 above)
Comment (cpmeister) [reply bundles/org.openhab.binding.miio/README.md:388]: realized you just moved the text. Nvm then.

**Comment by marcelrv** [comment]
sorry I missed your message. 
Indeed I did not change line ending. Instead  I moved the list of devices below the main text as I recognized many folks did not read any further. Hopefully now esp the FAQ is more easily found.

**Pull request merged**